parts/k8s/addons/1.16/kubernetesmasteraddons-cilium-daemonset.yaml

```diff
@@ -50,6 +50,7 @@ spec:
       annotations:
         prometheus.io/port: "9090"
         prometheus.io/scrape: "true"
+        scheduler.alpha.kubernetes.io/critical-pod: ""
```

Comment:
So these annotations were removed from the initial 1.16.0-alpha.1 PR, because at that time in the 1.16 release cycle they had been deprecated? And we're adding them back because that deprecation was reverted?

Reply:
Exactly.